mscore/playpanel.ui

```diff
@@ -20,7 +20,7 @@
    <bool>true</bool>
   </property>
   <property name="features">
-   <set>QDockWidget::AllDockWidgetFeatures</set>
```

Comment:
Qt 4.8 lists this as obsolete already in the documentation, Qt 6 will finally remove it